spec.html

```diff
@@ -41513,8 +41513,10 @@ <h1>Corrections and Clarifications in ECMAScript 2015 with Possible Compatibilit
 
 <emu-annex id="sec-additions-and-changes-that-introduce-incompatibilities-with-prior-editions">
   <h1>Additions and Changes That Introduce Incompatibilities with Prior Editions</h1>
-  <p><emu-xref href="#sec-tonumber-applied-to-the-string-type"></emu-xref>: In ECMAScript 2015, ToNumber applied to a String value now recognizes and converts |BinaryIntegerLiteral| and |OctalIntegerLiteral| numeric strings. In previous editions such strings were converted to *NaN*.</p>
   <p><emu-xref href="#sec-reference-specification-type"></emu-xref>: In ECMAScript 2015, Function calls are not allowed to return a Reference value.</p>
+  <p><emu-xref href="#sec-tonumber-applied-to-the-string-type"></emu-xref>: In ECMAScript 2015, ToNumber applied to a String value now recognizes and converts |BinaryIntegerLiteral| and |OctalIntegerLiteral| numeric strings. In previous editions such strings were converted to *NaN*.</p>
+  <p><emu-xref href="#sec-code-realms"></emu-xref>: In ECMAScript 2018, Template objects are canonicalized based on Parse Node (source location), instead of per Realm as in previous editions.</p>
```

Comment:
I'd phrase this as, "instead of by the template string [within a Realm]" (it's per Realm either way, in some sense; parse nodes are not shared between Realms).

Reply:
perhaps “instead of across all occurrences of that template string in a Realm”?